docs/alerting/email-destination.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: Destination type - Email
+nav_order: 1
+parent: monitors
+has_children: false
+---
+
+#  Email as a Destination 
+A destination is a location where you can send an alert that was triggered by the monitor. To send the alert as an email, select **Email** as the destination **Type**. You would need to configure at least one  **Sender** email address and add a **Recipient** email address. These two settings can be configured and managed using **Manage senders** and **Manage email groups**. 
+
+## Manage Senders
+Manage senders allows you to configure and manage **Sender** email addresses. Unless at least one sender is configured in the settings, the **Sender** dropdown list will be empty. 
```

Comment:
What is a sender? Why do I need one? What's an example?

Reply:
done